flake.nix

```diff
@@ -0,0 +1,60 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.05";
+    flake-compat = {
+      url = "github:edolstra/flake-compat";
+      flake = false;
+    };
+  };
+
+  outputs = { self, nixpkgs, flake-compat, ... }:
+    let
+      supportedSystems =
+        [ "x86_64-linux" "x86_64-darwin" "aarch64-linux" "aarch64-darwin" ];
+      forAllSystems = nixpkgs.lib.genAttrs supportedSystems;
+      nixpkgsFor = forAllSystems (system: import nixpkgs { inherit system; });
+    in {
+      packages = forAllSystems (system:
+        let
+          pkgs = nixpkgsFor.${system};
+          inherit (pkgs) stdenv lib;
+
+          propagatedBuildInputs = with pkgs;
+            with python3Packages; [
+              psutil
+              pwntools
+              yosys
+            ];
+        in rec {
+          default = verilog_tools;
+          verilog_tools = with pkgs;
+            with python3Packages;
+            buildPythonPackage rec {
+              pname = "verilog_tools-${version}";
+              version = "0.0.1";
+
+              format = "setuptools";
+              src = ./.;
+
+              inherit propagatedBuildInputs;
+
+              nativeCheckInputs = [ ruff black isort mypy ];
+              postCheck = ''
+                make format
+                make lint
+              '';
+            };
+        });
+
+      apps = forAllSystems (system: rec {
```

Comment:
Since there is no default app, we might consider adding some text to `README.md` providing some more detail about the need to invoke as e.g. `nix run .#sv-netlist`. Someone who is not familiar with flakes or conversant with our tooling might try `nix run` or `nix run github:trailofbits/verilog_tools` and be confused about an error like:
```
error: unable to execute '/nix/store/f6x4y82gg1sgj5l3d945viagwzmgrxlz-python3.10-verilog_tools-0.0.1-0.0.1/bin/verilog_tools-0.0.1': No such file or directory
```